src/java/main/org/apache/zookeeper/server/ServerCnxn.java

```diff
@@ -85,6 +86,14 @@ public boolean removeAuthInfo(Id id) {
         return authInfo.remove(id);
     }
 
+    public boolean isAuthCheckComplete() {
+        return authCheckCompleted;
+    };
```

Comment:
nit: please remove the semicolon after the close bracket

Reply:
will do.